html/elements/input/color.json

```
@@ -17,11 +17,11 @@
                 "version_added": "14"
               },
               "firefox": {
-                "version_added": "29",
-                "notes": "Firefox doesn't yet support inputs of type <code>color</code> on Windows Touch."
+                "version_added": "29"
               },
               "firefox_android": {
-                "version_added": "27"
+                "version_added": "27",
+                "notes": "Firefox on Android doesn't allow the user to choose a custom color, only one of the predefined ones."
```

nit: in most places we use "Firefox for Android" (though I'm sure there are inconsistencies that need cleaning up)

```suggestion
                "notes": "Firefox for Android doesn't allow the user to choose a custom color, only one of the predefined ones."
```